pkgs/deb/Makefile

```diff
@@ -0,0 +1,75 @@
+BUILDDIR=builddir
+URL=https://github.com/OpenMPDK/xNVMe/releases/download/v0.7.3/xnvme-0.7.3.tar.gz
+
+SRC_ROOT := $(BUILDDIR)/src
+TGZ_FILENAME := $(notdir $(URL))
+NAME := $(shell echo $(TGZ_FILENAME) | cut -d '-' -f 1)
+VERSION := $(shell echo $(TGZ_FILENAME) | cut -d '-' -f 2 | cut -d '.' -f 1-3)
```

Comment:
Perhaps it would be easier to set the version here and use that variable in the URL?

Reply:
This got a make-over as the download functionality was never used, what is handy is invoking "make gen-src-archive" when fixing up the repository when packaging, thus changed it to do so instead.